src/views/account/details/index.js

```diff
@@ -2,28 +2,53 @@ import React from 'react';
 import DetailsContent from './details';
 import Details from '../../../components/details';
 import {DataSource} from "../../../datasource";
+import { Alert } from "react-bootstrap";
 
 class AccountDetails extends React.Component {
 
     constructor(props) {
         super(props)
 
-        this.state = { id: 0, balance: 0, data: { public_key: "", coins: []}}
+        this.state = { id: 0, balance: 0, data: { public_key: "", coins: []}, showMessage: false}
         this.dataSource = DataSource.instance
         this.idAccount = this.props.location.pathname.replace("/account/", "")
     }
 
     componentWillMount() {
         this.dataSource.getAccount(this.idAccount).then(account => {
-            console.log(account)
-            if(account !== undefined)
-                this.setState({ id: account.id, balance: account.balance, data: account.data})
+            if(account !== undefined) {
+                this.setState({id: account.id, balance: account.balance, data: account.data})
+            } else {
+                this.setState({showMessage: true})
+            }
         })
     }
 
     render() {
+
+        function AlertMessage() {
+            return (
+                <Alert variant="danger" dismissible>
+                    <Alert.Heading>NO RESULT FOR THIS SEARCH!</Alert.Heading>
+                    <p>
+                        Try searching by Transaction hash, Block number or Account address
```

Comment:
Remove capital letter on transaction, block, and account

Reply:
Ok